workers/ohsome_quality_analyst/reports/jrc_requirements/report.py

```diff
@@ -1,23 +1,13 @@
-from typing import Optional
-
 from geojson import Feature
 
 from ohsome_quality_analyst.base.report import BaseReport, IndicatorLayer
 
 
 class JrcRequirements(BaseReport):
-    def __init__(
-        self,
-        feature: Feature = None,
-        dataset: Optional[str] = None,
-        feature_id: Optional[int] = None,
-        fid_field: Optional[str] = None,
-    ) -> None:
+    def __init__(self, feature: Feature = None) -> None:
         """Create a list of indicator objects."""
 
-        super().__init__(
-            feature=feature, dataset=dataset, feature_id=feature_id, fid_field=fid_field
-        )
+        super().__init__(feature=feature)
```

Comment:
I'm not sure, but isn't this the default constructor? (same applies to the other classes in `reports/*/report.py`)

Reply:
Your are right :) I removed the constructor for every report class.